app/jobs/sms_sender_confirmation_job.rb

```diff
@@ -4,7 +4,7 @@ class SmsSenderConfirmationJob < ActiveJob::Base
   def perform(code, mobile)
     TwilioService.new.send_sms(
       to: mobile,
-      body: "Your #{APP_NAME} phone confirmation code is: #{code}."
+      body: "#{code} is your #{APP_NAME} one-time passcode."
```

Comment:
the colon feels a bit awkward to me; i'd prefer to leave as is and throw it over the fence for research team / user testing.